cpp/modmesh/python/wrapper/modmesh/wrap_SimpleArray.cpp

```diff
@@ -146,6 +146,17 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                 "__setitem__",
                 [](wrapped_type & self, std::vector<ssize_t> const & key, T val)
                 { self.at(key) = val; })
+            .def(
+                "__setitem__",
+                [](wrapped_type & self, const py::ellipsis &, pybind11::array & arr_in)
+                {
+                    auto arr_in_size = static_cast<size_t>(arr_in.nbytes());
+                    if (self.nbytes() != arr_in_size)
+                    {
+                        throw std::out_of_range("Array sizes mismatch");
```

Comment:
The error message should include the mismatch shapes, including the input and self.

Reply:
done